static/app/views/issueDetails/shortIdBreadcrumb.tsx

```diff
@@ -126,6 +126,8 @@ const Wrapper = styled('div')`
   display: flex;
   gap: ${space(1)};
   align-items: center;
+  /* hardcoded height avoids layout shift on button hover */
+  height: 36px;
```

Comment:
hmm this doesn’t fix it for me, are you sure this is the right place?

I think it needs to go to this Wrapper:

https://github.com/getsentry/sentry/blob/aff87106891abd5aaa46cbf4e384d531bc9d1249/static/app/views/issueDetails/streamline/header/issueIdBreadcrumb.tsx#L122-L125

to have an effect on the issue details page. Maybe it’s different for different issues, don’t know. This is the one I was testing:

```
issues/6557993067/?project=1&query=is%3Aunresolved%20issue.priority%3A%5Bhigh%2C%20medium%5D&referrer=issue-stream&stream_index=1
```

Reply:
Good catch! No idea how this got mixed up when it seemed to be working locally, but can confirm that [777dd16](https://github.com/getsentry/sentry/pull/90953/commits/777dd16b5aedb7bbf759fe31516f5edaf6114f94) is the right fix